### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by HichamChaaya** [psi-4ward/psitransfer#79]
Is it possible to add "Delete file" in the admin console?

**Comment by psi-4ward** [comment]
No. Currently you have to go to your filesystem, delete the buckets and then restart PsiTransfer.

**Comment by digitalentity** [comment]
+1 to this feature request!

**Comment by luckman212** [comment]
This would be so useful

**Comment by CyBuzz** [comment]
I would say this is +1 more for the feature but deleting from the FS and restarting is easy enough.  Thanks for your hard work on this tool.

**Comment by televisi** [comment]
+1 to this useful feature

**Comment by JanC** [comment]
+1 :)
